rest-service/manager_rest/storage/storage_manager.py

```diff
@@ -36,7 +40,12 @@
 
 
 class SQLStorageManager(object):
+    @with_tracing()
+    def __init__(self):
+        self._enable_tracing()
+
     @staticmethod
+    @with_tracing()
```

Comment:
why not use the wrapper?

Reply:
`sqlalchemy` wrapper? Because it doesn't use `threading.local()` which what I implemented is based on.